packages/dcos-ui/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-ui.git",
-    "ref": "b51934df009bd2a86dec605cdc5b34f006feffef",
-    "ref_origin": "master"
+    "ref": "e79da607f632ad36c5f19c9a7890ab15207b9f74",
+    "ref_origin": "feature/system-integration-tests"
```

Comment:
I'm hesitant to point at any feature branch as these are usually short lived.
Can we wait until this merges to `master`?